fabric-renderer-indigo/src/main/java/net/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl.java

```diff
@@ -81,6 +81,11 @@ public final MutableQuadViewImpl cullFace(Direction face) {
 		return this;
 	}
 
+	/*
```

Comment:
```suggestion
	/**
```
Since this is a javadoc comment.

Reply:
Comments was removed entirely